#object info
int32 id #object id
string name #object name

#object class
int32 class_id
string class_name

#bounding box[m]
geometry_msgs/Vector3 dimensions
int32 kalman_account